src/main/scala/core/MasterActor.scala

```diff
@@ -1,35 +1,30 @@
-package main
+package core
 
 import akka.actor.{Actor, ActorLogging, ActorRef, PoisonPill, Props, Terminated}
 import ch.qos.logback.classic.LoggerContext
 import ch.qos.logback.core.util.StatusPrinter
-import database.DatabaseActor
 import org.slf4j.LoggerFactory
-import scheduler.Basched
 import utils.Configuration
-import webserver.WebServerActor
 
 import scala.collection.mutable.ArrayBuffer
 
 /**
   * Created by igor on 10/05/17.
   */
-object MasterActor {
-  def props(config: Configuration): Props = Props(new MasterActor(config))
-}
 
 class MasterActor(config: Configuration) extends Actor with ActorLogging {
 
   // Print the status of the logger.
-  val lc = LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
+  val lc: LoggerContext = LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
```

Comment:
Why you need to know the type of the variable here?

Reply:
IntelliJ recommendation - i can revert if you would like